### ISSUE TITLE
LSCache assumes REST API prefix is wp-json

### ISSUE THREAD

**Issue by therealgilles** [litespeedtech/lscache_wp#174]
The code should query the prefix using rest_get_url_prefix() instead of hardcoding 'wp-json'.

**Comment by hailite** [comment]
Good catch! 

Fixed in https://github.com/litespeedtech/lscache_wp/commit/73f18e93b54061a092c23d361f51ba4a373fd134 

Thanks again for your report.

**Comment by therealgilles** [comment]
I think you missed a couple:

```
> find wp-content/plugins/litespeed-cache -type f -exec grep wp-json {} \; -print
		if ( strpos( $_SERVER[ 'REQUEST_URI' ], '/wp-json/contact-form-7' ) !== false ) {
wp-content/plugins/litespeed-cache/thirdparty/lscwp-3rd-contact-form-7.cls.php
		// Case #3: URL Path begins with wp-json/ (REST prefix) Safe for subfolder installation
wp-content/plugins/litespeed-cache/inc/rest.cls.php
			strpos( $_SERVER[ 'REQUEST_URI' ], 'wp-json/wp/v2/media' ) !== false
			LiteSpeed_Cache_Log::debug( '[Router] CDN bypassed: wp-json on admin page' ) ;
wp-content/plugins/litespeed-cache/inc/router.class.php
			strpos( $_SERVER[ 'REQUEST_URI' ], 'wp-json/wp/v2/media' ) !== false
			LiteSpeed_Cache_Log::debug( '[Router] CDN bypassed: wp-json on admin page' ) ;
wp-content/plugins/litespeed-cache/includes/litespeed-cache-router.class.php
```

**Comment by hailite** [comment]
Oh apologized for missing mentioned that the fix is only on v3.0 branch. 

Next time when merging master to v3.0, will do. Thanks for the notes.

**Comment by hailite** [comment]
The whole repo are corrected. Also appended the parameter to online request in https://github.com/litespeedtech/lscache_wp/commit/05a5fec34f4cf423eb090d27ed4f63fefd5fba22 in v3.0 branch.
